DeathNotes.cs

```diff
@@ -425,7 +435,8 @@ internal enum CombatEntityType
             Lock = 12,
             ScientistSentry = 13,
             Other = 14,
-            None = 15
+            None = 15,
+            TunnelDweller = 17

```

Comment:
I would move the TunnelDweller entry to below the Scarecrow entry perhaps, as it makes a bit more sense there and to keep Other and None at the bottom.

It also looks like the TunnelDweller should have a switch-case in GetEntityName, returning "Tunnel Dweller".